ballerina/modules/types.any/any.bal

```diff
@@ -60,11 +60,12 @@ public type ContextAnyStream record {|
 # Generate and return the generic `'any:Any` record that is used to represent protobuf `Any` type.
 #
 # + message - The record or the scalar value to be packed as Any type
-# + return - Any value representation of the given message
-public isolated function pack(ValueType message) returns Any {
+# + return - Any value representation of the given message or an error
```

Comment:
```suggestion
# + return - Any value representation of the given message or, an Error
```

Reply:
I thought of not highlighting the customised `Error`. Instead, I just wanted to use error as a text.